src/events/multi-asset.upsert.service.ts

```diff
@@ -35,12 +35,21 @@ export class MultiAssetUpsertService {
         operation.type === BlockOperation.CONNECTED ||
         operation.type === BlockOperation.DISCONNECTED;
 
-      if (shouldUpsertMultiAsset) {
+      if (
+        shouldUpsertMultiAsset &&
+        this.occurredAfterLaunch(operation.block.timestamp)
+      ) {
         await this.upsert(operation);
       }
     }
   }
 
+  occurredAfterLaunch = (comparisonDate: Date): boolean => {
+    const timestamp = this.config.get<string>('PHASE3_LAUNCH_TIMESTAMP');
+    const launch = new Date(+timestamp);
```

Comment:
Can we just use `.get<number>`? Would that work?

Reply:
I put it as that previously, but it actually returns a string, so I changed it to this to match actual type. Was getting an invalid date because the value was actually a string (even though we told type system it was returning a number).